src/GraphQL.Tests/Utilities/NameValidatorTests.cs

```diff
@@ -23,14 +23,22 @@ public class NameValidatorTests
         public void ValidateName_whenNameStartsWithReservedCharacters_throwsArgumentOutOfRange() =>
             Should.Throw<ArgumentOutOfRangeException>(() => ValidateName("__dede"));
 
-        [Fact]
-        public void ValidateName_whenNameContainsInvalidCharacters_throwsArgumentOutOfRange() =>
-            Should.Throw<ArgumentOutOfRangeException>(() => ValidateName("śćłó"));
+        [Theory]
+        [InlineData("śćłó")]
+        [InlineData("3test")]
+        [InlineData("test Name")]
+        public void ValidateName_whenNameContainsInvalidCharacters_throwsArgumentOutOfRange(string invalidName) =>
+            Should.Throw<ArgumentOutOfRangeException>(() => ValidateName(invalidName));
 
-        [Fact]
-        public void ValidateName_whenNameIsCorrect_DoesntthrowsArgumentOutOfRange()
+        [Theory]
+        [InlineData("goodName")]
+        [InlineData("name3")]
+        [InlineData("Test")]
+        [InlineData("Test_Name")]
+        [InlineData("_test")]
+        public void ValidateName_whenNameIsCorrect_DoesntthrowsArgumentOutOfRange(string validName)
         {
-            var ex = Record.Exception(() => ValidateName("goodName"));
+            var ex = Record.Exception(() => ValidateName(validName));
             ex.ShouldBeNull();
```

Comment:
```suggestion
            ValidateName(validName);
```

Reply:
Also I'm not abig fan of `using static` because it's not immediately clear where the method came from. There is little savings, but questions arise.

Reply:
I agree, and I also agree with this change.  I was just copying what was in there... I'll fix it.